sample-job-handlers/verify-packages-installed.sh

```diff
@@ -8,28 +8,29 @@ packages=$@
 echo "Username: $user"
 echo "Packages to verify: $packages"
 
-if command -v "rpm" > /dev/null; then
+if id "$user" 2>/dev/null && command -v "sudo" > /dev/null; then
   for package in $packages
   do
-    if id "$user" 2>/dev/null && command -v "sudo" > /dev/null; then
-      sudo -u "$user" -n rpm -q "$package"
-    else
-      echo "username or sudo command not found"
-      rpm -q "$package"
+    if command -v "rpm" >/dev/null; then
+      if command sudo -u "$user" -n rpm -q "$package" > /dev/null; then
```

Comment:
why do we have a conditional check over here? we are not doing anything if it returns either `true` or `false`.

Reply:
If true we move on to the next package, if false we check dpkg. I did forgot to exit 1 though if both fail... will fix.